docs/user-guide/home/global-modules/README.md

```diff
@@ -0,0 +1,134 @@
+
+![openproject_user_guide_global_team_planners](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_team_planner.png)
+
+You can also create a new team planner directly from the global modules menu by clicking the green **+Team planner** button. 
+
+![openproject_user_guide_add_new_team_planner](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_team_planner.png)
+
+Here you can name the team planner, select a project to which a team planner should be assigned to and set to be public or favored. Find out more about editing team planners [here](../../team-planner). 
+
+## Boards
+
+**Boards** global module will list all boards that have been created in the projects you are a member of, have administrative privileges to see or public projects. 
+
+![openproject_user_guide_global_boards](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_boards.png)
+
+You can also create a board directly from the global modules menu by clicking the green **+Board** button. 
+
+![openproject_user_guide_add_new_board](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_add_board.png)
+
+Here you can name the board, select a project to which a board should be assigned to and choose the board type. Find out more about editing boards [here](../../agile-boards). 
+
+## News
+
+**News** global module will display all news that have been published in projects you are a member of, have administrative privileges to see and public projects. 
+
+TIMEFRAME - ?
+
+![openproject_user_guide_global_news](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_news.png)
+
+Read more about writing, editing and commenting on **News** in OpenProject [here](../../news).
+
+## Time and costs
+
+**Time and costs** global module will list time and cost reports created by you and the ones set to be public in the left hand menu. The filters you see initially will be pre-filled based on the values you used for filtering during your last session. You can create new time and cost reports by adjusting filters, grouping criteria and units accordingly and clicking **Apply** button. Do not forget to **Save** your report. 
+
+![openproject_user_guide_global_time_cost](C:\Users\Maya\Documents\GitHub\openproject\docs\user-guide\home\global-modules\openproject_global_modules_time_costs.png)
+
+Read more about creating and editing **Time and cost reports** [here](../../time-and-costs/reporting/).
+
+## Meetings
+
+**Meetings** global module will provide a list of all upcoming an past meetings you have created, been invited to, attended or have administrative privileges to see. The default view will show the **Upcoming meetings**. The most current meetings are shown on top of the list. 
```

Comment:
I don't think "the most current meetings" is clear, maybe "the closest meetings"?

Reply:
I changed this in my commit to: "The default view will show the **Upcoming meetings**, in chronological order (earliest first)."